mpqp/noise/noise_model.py

```diff
@@ -142,6 +145,7 @@ class Depolarizing(NoiseModel):
              └───┘
         NoiseModel:
             Depolarizing(0.32, [0, 1, 2], 1)
+            Depolarizing(0.01, [all], 1)
```

Comment:
Why `all` in the array ?

Reply:
there are several solutions for print:
 - execute hard_copy to obtain the real target at step
 - print 'all' or 'dynamic', to show that it's for all qubits, so don't parse all instructions